test/unittests/execute_floating_point_test.cpp

```diff
@@ -100,9 +117,71 @@ struct WasmTypeName
     }
 };
 
+template <typename T>
+class execute_floating_point_types : public testing::Test
+{
+protected:
+    /// Creates a wasm module with a single function for the given instructions opcode.
+    /// The opcode is converted to match the type, e.g. f32_add -> f64_add.
+    bytes get_binop_code(Instr opcode)
```

Comment:
Nit: why not mark this static in this commit?

Reply:
Fixed.